### ISSUE TITLE
Cannot disable UseIfInsteadOfWhen

### ISSUE THREAD

**Issue by andrey-bolduzev** [arturbosch/detekt#2007]
## Expected Behavior
Having:
```
style:
...
  UseIfInsteadOfWhen:
    active: false
...
```
in the detekt config should disable `UseIfInstead` inspection in the project.

## Observed Behavior
The inspection is being run and reported.

## Steps to Reproduce
The relevant part of the gradle groovy config:

```groovy
detekt {
	version = Versions.detekt_api
	config = project.parent.files('.static/config/default-detekt-config.yml')
	baseline = project.parent.file('.static/config/baseline.xml')
	input = files(
			"src/main/java",
			"src/test/java",
			"src/androidTest/java",
			"src/commonTest/java",
			"src/debug/java",
			"src/release/java"
	)
	parallel = true
	failFast = false
	buildUponDefaultConfig = true

	reports {
		html {
			enabled = true
			destination = project.file("build/reports/detekt_${project.name}.html")
		}
		xml.enabled = false
	}
}
```

## Context
After updating to detekt 1.1.0, this is literally the only inspection that ignores the setting in the config file.

## Your Environment
* Version of detekt used: 1.1.0

**Comment by BorzdeG** [comment]
duplicate #1986

**Comment by schalkms** [comment]
This has been fixed already. Version 1.1.1 contains this fix and will be released in the next days.

**Comment by andrey-bolduzev** [comment]
Yep, sorry, my bad for not looking more carefully in the closed issues.

**Comment by schalkms** [comment]
No worries. @andrey-bolduzev
